examples/example_9_1_Bathe.jl

```diff
@@ -1,60 +1,57 @@
 # # Example (Ch. 9 Bathe)
 
-#md # [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/models/example_9_1_Bathe.ipynb)
```

Comment:
This is throwing some errors, I don't find value to fix it right now. If you are keen feel free to tackle it or we can create a good first issue to migrate to Pluto and deploy those directly.